### PR TITLE
[14.0][MIG] purchase_delivery_split_date

### PR DESCRIPTION
**Pull request by SurachetS**

### CODE REVIEW

File: purchase_delivery_split_date/readme/HISTORY.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+12.0.2.0.0 (2020-04-10)
```
Comment (kittiu) [inline purchase_delivery_split_date/readme/HISTORY.rst:1]: Should you add your MIG in history too.

**Review by kittiu — APPROVED** [review]
Functional tested, LGTM

**Review by atchuthan — APPROVED** [review]
code :+1:

**Pull request merged**